esphome/components/ballu_old/climate.py

```diff
@@ -0,0 +1,21 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import climate_ir
+from esphome.const import CONF_ID
+
+AUTO_LOAD = ["climate_ir"]
+CODEOWNERS = ["@Midnighter32"]
+
+ballu_old_ns = cg.esphome_ns.namespace("ballu_old")
+BalluOldClimate = ballu_old_ns.class_("BalluOldClimate", climate_ir.ClimateIR)
+
+CONFIG_SCHEMA = climate_ir.CLIMATE_IR_WITH_RECEIVER_SCHEMA.extend(
```

Comment:
There is no `on_receive` code for the receiver...
```suggestion
CONFIG_SCHEMA = climate_ir.CLIMATE_IR_SCHEMA.extend(
```